androidtv/basetv/basetv.py

```diff
@@ -128,6 +128,9 @@ def _cmd_audio_state(self):
         if constants.CUSTOM_AUDIO_STATE in self._custom_commands:
             return self._custom_commands[constants.CUSTOM_AUDIO_STATE]
 
+        # Is this a Android 11 Android TV?
+        if self.DEVICE_ENUM == constants.DeviceEnum.ANDROIDTV and self.device_properties.get("sw_version", "") == 11:
```

Comment:
```suggestion
        if self.DEVICE_ENUM == constants.DeviceEnum.ANDROIDTV and self.device_properties.get("sw_version", "") == "11":
```

Please update the other occurrences, as well.